account/recovery.go

```diff
@@ -123,17 +128,254 @@ func recoverInitalState(ctx context.Context, cfg RecoveryConfig) (
 	return accounts, nil
 }
 
+// recreatePossibleAccounts returns a set of potentially valid accounts
+// by generating a set of recovery keys.
+func recreatePossibleAccounts(ctx context.Context, cfg RecoveryConfig) (
+	map[*Account]struct{}, error) {
+
+	possibleAccounts := make(map[*Account]struct{}, cfg.AccountTarget)
+
+	// Prepare the keys we are going to try. Possibly not all of them will
+	// be used.
+	KeyDescriptors, err := GenerateRecoveryKeys(
+		ctx, cfg.AccountTarget, cfg.Wallet,
+	)
+	if err != nil {
+		return possibleAccounts, fmt.Errorf(
```

Comment:
nit: `Errorf()` and `Debugf()` calls should use the compact formatting to better distinguish against normal function/method calls.